ios/keyman/Keyman/Keyman/SetUpViewController/SetUpViewController.swift

```diff
@@ -44,12 +44,15 @@ class SetUpViewController: UIViewController, UIWebViewDelegate {
   }
 
   func reloadKeymanHelp() {
-    let networkStatus = networkReachable?.connection
-    switch networkStatus {
-    case Reachability.Connection.none?:
+    if let networkStatus = networkReachable?.connection {
+      switch networkStatus {
+      case Reachability.Connection.none, Reachability.Connection.unavailable:
+        loadFromLocal()
+      default:
+        loadFromServer()
+      }
+    } else {
```

Comment:
I can see a case for always loading help from local now that we have it locally -- that means the help will be in sync with the product. But no need to make that change just now.